sdk/ml/azure-ai-ml/azure/ai/ml/constants/_component.py

```diff
@@ -54,11 +54,9 @@ class ExternalDataType(object):
 
 
 class DataTransferBuiltinComponentUri(object):
-    # todo: need update
-    IMPORT_DATABASE = "azureml://registries/import_database"
-    IMPORT_FILE_SYSTEM = "azureml://registries/import_file_system"
-    EXPORT_DATABASE = "azureml://registries/export_database"
-    EXPORT_FILE_SYSTEM = "azureml://registries/export_file_system"
+    IMPORT_DATABASE = "azureml://registries/azureml-dev/components/import_data_database/versions/0.0.1"
```

Comment:
do AML support labels/latest now?

Reply:
Not sure, only give me these three registry component ids for now.